interview.md

```diff
@@ -58,11 +78,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I belive most of this project would be spend on the development of the language  and less so on the development of the system. This project whould have to optimize user understanging of the language as well as user speed, and language difficulty. 
+
```

Comment:
I agree - the design and streamlining of the interaction would be the most difficult part.